site/docs/4.5.1/overview/releaseNotes.md

```diff
@@ -11,491 +11,16 @@ below.
-It is simple but not flexible on ledger id or entry id generation. Apache BookKeeper introduces `LedgerHandleAdv`
-as an extension of existing `LedgerHandle` for advanced usage. The new `LedgerHandleAdv` allows applications providing
-its own `ledger-id` and assigning `entry-id` on adding entries.
-
-See [Ledger Advanced API](../../api/ledger-adv-api) for more details.
-
-#### Long Poll
-
-`Long Poll` is a main feature that [DistributedLog](https://distributedlog.io) uses to achieve low-latency tailing.
-This big feature has been merged back in 4.5.0 and available to BookKeeper users.
-
-This feature includes two main changes, one is `LastAddConfirmed` piggyback, while the other one is a new `long poll` read API.
-
-The first change piggyback the latest `LastAddConfirm` along with the read response, so your `LastAddConfirmed` will be automatically advanced
-when your read traffic continues. It significantly reduces the traffic to explicitly polling `LastAddConfirmed` and hence reduces the end-to-end latency.
-
-The second change provides a new `long poll` read API, allowing tailing-reads without polling `LastAddConfirmed` everytime after readers exhaust known entries.
-Although `long poll` API brings great latency improvements on tailing reads, it is still a very low-level primitive.
-It is still recommended to use high level API (e.g. [DistributedLog API](../../api/distributedlog-api)) for tailing and streaming use cases.
-
-See [Streaming Reads](https://distributedlog.incubator.apache.org/docs/latest/user_guide/design/main.html#streaming-reads) for more details.
-
-#### Explicit LAC
-
-Prior to 4.5.0, the `LAC` is only advanced when subsequent entries are added. If there is no subsequent entries added,
-the last entry written will not be visible to readers until the ledger is closed. High-level client (e.g. DistributedLog) or applications
-has to work around this by writing some sort of `control records` to advance `LAC`.
-
-In 4.5.0, a new `explicit lac` feature is introduced to periodically advance `LAC` if there are not subsequent entries added. This feature
-can be enabled by setting `explicitLacInterval` to a positive value.
-
-### Performance
-
-There are a lot for performance related bug fixes and improvements in 4.5.0. These changes includes:
-
-- Upgraded netty from 3.x to 4.x to leverage buffer pooling and reduce memory copies.
-- Moved developement from Java 7 to Java 8 to take advantage of Java 8 features.
-- A lot of improvements around scheduling and threading on `bookies`.
-- Delay ensemble change to improve tail latency.
-- Parallel ledger recovery to improve the recovery speed.
-- ...
-
-We outlined following four changes as below. For a complete list of performance improvements, please checkout the `full list of changes` at the end.
+- Fix critical bug on Parallel Recovery, see [https://github.com/apache/bookkeeper/issues/343](https://github.com/apache/bookkeeper/issues/343)
```

Comment:
there are also a few others:

- ledger length in ledgerhandleadv
- rack aware placement policy

Reply:
done